src/cljs/main/broadfcui/auth.cljs

```diff
@@ -170,9 +168,10 @@
               (links/create-external {:style {:display "block" :marginTop "0.3rem"}
                                       :href "https://software.broadinstitute.org/firecloud/documentation/article?id=9846"}
                 "Learn how to create a Google account with any email address.")])]
-          [:div {:style {:width 180 :paddingLeft "2rem" :alignSelf "center"}
-                 :className "g-signin2" :data-theme "dark" :data-width 180 :data-height 40 :data-longtitle true
-                 :onClick #(this :-handle-sign-in-click)}]]]
+          [:div {:id "sign-in-button"
+                 :style {:flexShrink 0 :width 180 :paddingLeft "2rem" :alignSelf "center"}
```

Comment:
You should only need one of flex-shrink and width. I'd vote for the former.

Reply:
It definitely needs both here. Width alone doesn't keep flex from trying to shrink it, and without width the size jumps to the child's size.

Reply:
Huh. Cool.